src/Kernel32.Desktop/Kernel32.cs

```diff
@@ -68,6 +73,52 @@ public static partial class Kernel32
+        /// Initially, this list contains only a default handler function that calls <see cref="ExitProcess"/>.
+        /// A console process adds or removes additional handler functions by calling the <see cref="SetConsoleCtrlHandler"/> function,
+        /// which does not affect the list of handler functions for other processes. When a console process receives any of the control signals,
+        /// its handler functions are called on a last-registered, first-called basis until one of the handlers returns TRUE.
+        /// If none of the handlers returns TRUE, the default handler is called.
+        /// </para>
+        /// <para>
+        /// The <see cref="ControlType.CTRL_CLOSE_EVENT"/>, <see cref="ControlType.CTRL_LOGOFF_EVENT"/>, and <see cref="ControlType.CTRL_SHUTDOWN_EVENT"/> signals give the process
+        /// an opportunity to clean up before termination. A <see cref="PHANDLER_ROUTINE"/> can perform any necessary cleanup, then take one of the following actions:
+        /// </para>
+        /// <list>
+        /// <item>Call the <see cref="ExitProcess"/> function to terminate the process.</item>
+        /// <item>Return FALSE. If none of the registered handler functions returns TRUE, the default handler terminates the process.</item>
+        /// <item>Return TRUE. In this case, no other handler functions are called and the system terminates the process.</item>
+        /// </list>
+        /// <para>
+        /// A process can use the <see cref="SetProcessShutdownParameters"/> function to prevent the system from displaying a dialog box to the user during logoff or shutdown.
+        /// In this case, the system terminates the process when <see cref="PHANDLER_ROUTINE"/> returns TRUE or when the time-out period elapses.
+        /// When a console application is run as a service, it receives a modified default console control handler.
+        /// This modified handler does not call <see cref="ExitProcess"/> when processing the <see cref="ControlType.CTRL_LOGOFF_EVENT"/> and <see cref="ControlType.CTRL_SHUTDOWN_EVENT"/> signals.
+        /// This allows the service to continue running after the user logs off.
+        /// If the service installs its own console control handler, this handler is called before the default handler.
+        /// If the installed handler calls <see cref="ExitProcess"/> when processing the <see cref="ControlType.CTRL_LOGOFF_EVENT"/> signal, the service exits when the user logs off.
+        /// </para>
+        /// <para>
+        /// Note that a third-party library or DLL can install a console control handler for your application.
+        /// If it does, this handler overrides the default handler, and can cause the application to exit when the user logs off.
+        /// </para>
+        /// </remarks>
+        [UnmanagedFunctionPointer(CallingConvention.Winapi)]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        public unsafe delegate bool PHANDLER_ROUTINE(ControlType dwCtrlType);
```

Comment:
On the native side this is defined as `HandlerRoutine`, where a dedicated pointer type for it is `PHANDLER_ROUTINE`. So I think this should be renamed to `HandlerRoutine`.
Why is this marked `unsafe`?